olla/include/openmm/Platform.h

```diff
@@ -227,6 +227,13 @@ class OPENMM_EXPORT Platform {
      * OPENMM_PLUGIN_DIR is set, this returns its value.  Otherwise, it returns a platform
      * specific default location.
      *
+     * On unix-like systems, OPENMM_PLUGIN_DIR is allowed to either be a
```

Comment:
This should go in the documentation for `loadPluginsFromDirectory()`, not `getDefaultPluginsDirectory()`.  Any path you tell it to load from can be a set of multiple directories, not just the default one.

Reply:
Sorry, that was a typo on my part. I was parsing the comment as applying to whatever it was below, not what it was above.